06-working-w-copilot/copilot-interface-shortcuts.md

```diff
@@ -0,0 +1,223 @@
+*Fig. A comment describing a function where Copilot is suggesting to add another commented line*
+
+We're going to travel back in time to our first readings about LLMs. There, we learned that LLMs are, in many ways, incredibly powerful predictive text engines. When our cursor is on a line with a comment, Copilot has determined that the most likely thing we're going to do is continue writing the comment, so that's what it suggests. It isn't until we've created a few new lines from the comment that Copilot will reach a probability that we want code rather than descriptive information. Let's continue with this example until we see some code!
+
+When we create another new line, copilot tries to generate example inputs and outputs for us as comments. These comment additions can be really handy for thinking about edge cases we missed or tests we may need to write, feel free to try it out and see what examples you get.   
+![A comment in a python file describing a function where Copilot is suggesting to add an example to the comment](assets/copilot-interface-shortcuts/copilot-comment-prompt-example-text.png)  
+*Fig. Copilot suggesting to add an example to the comment*
+
+![A comment in a python file describing a function where Copilot has added an example input and output to the comment](assets/copilot-interface-shortcuts/copilot-comment-prompt-example-completed.png.png)  
+*Fig. An example input and output suggested by Copilot*
+
+If create yet one more new line, that is when copilot realizes we aren't trying to add any further description and provides us with a function definition. Once we press `Tab` to accept the function definition suggested, Copilot will suggest an implementation for the function.  
+![A comment in a python file describing a function where Copilot has then suggested a function definition](assets/copilot-interface-shortcuts/copilot-comment-prompt-function-definition.png)  
+*Fig. A function definition suggested by Copilot for the function described in our comment*
+
+![A comment in a python file describing a function and the function definition itself followed by a suggestion from Copilot for the body of the function](assets/copilot-interface-shortcuts/copilot-comment-prompt-full-function.png)  
+*Fig. A function body suggested by Copilot for the function described in our comment*
+
+### Asking in a chat
+
+Copilot has a chat feature where you can write out questions or code prompts to Copilot, similarly to how we've interacted with ChatGPT for code. We'll see how we can use it to create a function, walk us through how the function works, and help us test it.
+
+#### Starting a chat
+
+There are 2 ways we can interact with the Copilot chat:
+
+1. We can right click inside a file and select "Copilot > Start Inline Chat" or use the shortcut  `CMD + i` to open up a text box while we're still in a file.  
```

Comment:
As I recall they were separate while Chat was still in beta and it wasn't available to everyone with Copilot. I'm curious why they're still shown separately at this point. It doesn't hurt to call that out in case anyone might be confused.